### ISSUE TITLE
 unplugin-svg-component@0.7.3" has incorrect peer dependency "@vue/compiler-sfc@>=2.7.0 <=3.2.47".

### ISSUE THREAD

**Issue by thisVioletHydra** [Jevon617/unplugin-svg-component#7]
```q
warning " > unplugin-svg-component@0.7.3" has incorrect peer dependency "@vue/compiler-sfc@>=2.7.0 <=3.2.47".
```
is there an update planned?

**Comment by Jevon617** [comment]
I tested in my environment without any warning, could you provide a minimum reproduction?

**Comment by thisVioletHydra** [comment]
```cmd
yarn upgrade v1.22.19
[1/5] Validating package.json...
[2/5] Resolving packages...
warning unplugin-svg-component > svgo > stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
warning unplugin-svg-component > svg-baker > micromatch > snapdragon > source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated
warning unplugin-svg-component > svg-baker > micromatch > snapdragon > source-map-resolve > resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
warning unplugin-svg-component > svg-baker > micromatch > snapdragon > source-map-resolve > urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
warning unplugin-svg-component > svg-baker > micromatch > snapdragon > source-map-resolve > source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
warning vite-plugin-dts > @rushstack/node-core-library > z-schema@5.0.6: has issues with node 14
warning vite-plugin-fonts@0.7.0: renamed to `unplugin-fonts`, see https://github.com/cssninjaStudio/unplugin-fonts/releases/tag/v1.0.0
[3/5] Fetching packages...
warning vscode-languageclient@7.0.0: The engine "vscode" appears to be invalid.
[4/5] Linking dependencies...
warning " > postcss-normalize@10.0.1" has unmet peer dependency "browserslist@>= 4".
warning "postcss-normalize > postcss-browser-comments@4.0.0" has unmet peer dependency "browserslist@>=4".
warning " > stylelint-config-prettier@9.0.5" has incorrect peer dependency "stylelint@>= 11.x < 15".
warning " > unplugin-svg-component@0.7.3" has incorrect peer dependency "@vue/compiler-sfc@>=2.7.0 <=3.2.47".
warning " > vite-plugin-vue-css-modules@3.1.2" has unmet peer dependency "@vue/shared@^3.2.47".
```
i dont know how
take my package.json

```json
 "dependencies": {
    "@csstools/postcss-media-minmax": "^1.0.2",
    "@double-great/stylelint-a11y": "^2.0.2", 
    "@intlify/unplugin-vue-i18n": "^0.11.0",
    "@jridgewell/sourcemap-codec": "^1.4.15",
    "@prantlf/jsonlint": "^14.0.3",
    "@rushstack/eslint-patch": "^1.3.1",
    "@typescript-eslint/eslint-plugin": "5.60.1",
    "@typescript-eslint/parser": "5.60.1",
    "@vitejs/plugin-vue": "^4.2.3",
    "@vitest/coverage-c8": "^0.32.0",
    "@vue/compiler-sfc": "^3.3.4",
    "@vue/eslint-config-prettier": "^7.1.0",
    "@vue/eslint-config-typescript": "^11.0.3",
    "@vue/test-utils": "^2.4.0",
    "@vueuse/core": "^10.2.0",
    "auto-changelog": "^2.4.0",
    "autoprefixer": "^10.4.14",
    "axios": "^1.4.0",
    "axios-mock-adapter": "^1.21.4",
    "cssnano": "^6.0.1",
    "del-cli": "^5.0.0",
    "esbuild": "^0.18.10",
    "eslint": "^8.41.0",
    "eslint-config-airbnb-base": "^15.0.0",
    "eslint-config-airbnb-typescript": "^17.0.0",
    "eslint-config-prettier": "^8.8.0",
    "eslint-import-resolver-alias": "^1.1.2",
    "eslint-import-resolver-custom-alias": "^1.3.2",
    "eslint-import-resolver-typescript": "^3.5.5",
    "eslint-plugin-import": "^2.26.0",
    "eslint-plugin-json": "^3.1.0",
    "eslint-plugin-node": "^11.1.0",
    "eslint-plugin-package-json": "^0.1.4",
    "eslint-plugin-prettier": "^4.2.1",
    "eslint-plugin-prettier-vue": "^4.2.0",
    "eslint-plugin-promise": "^6.1.1",
    "eslint-plugin-regexp": "^1.15.0",
    "eslint-plugin-sonarjs": "^0.19.0",
    "eslint-plugin-unicorn": "^47.0.0",
    "eslint-plugin-vue": "^9.15.1",
    "jsdom": "^22.1.0",
    "jsonlint-mod": "^1.7.6",
    "npm-run-all2": "^6.0.5",
    "pinia": "^2.1.3",
    "postcss": "^8.4.24",
    "postcss-at-rules-variables": "^0.3.0",
    "postcss-calc": "^9.0.1",
    "postcss-combine-duplicated-selectors": "^10.0.3",
    "postcss-custom-properties": "^13.1.5",
    "postcss-discard-comments": "^6.0.0",
    "postcss-each": "^1.0.0",
    "postcss-for": "^2.1.1",
    "postcss-hexrgba": "^2.1.0",
    "postcss-html": "^1.5.0",
    "postcss-import": "^15.1.0",
    "postcss-logical": "^6.2.0",
    "postcss-merge-rules": "^6.0.1",
    "postcss-mixins": "^9.0.4",
    "postcss-nested": "^6.0.0",
    "postcss-normalize": "^10.0.1",
    "postcss-normalize-repeat-style": "^6.0.0",
    "postcss-normalize-timing-functions": "^6.0.0",
    "postcss-pxtorem": "^6.0.0",
    "postcss-rename": "^0.6.1",
    "postcss-reporter": "^7.0.4",
    "postcss-responsive-type": "^1.0.0",
    "postcss-simple-vars": "^7.0.1",
    "postcss-sorting": "^8.0.2",
    "postcss-url": "10.1.3",
    "prettier": "^2.8.8",
    "rimraf": "^5.0.1",
    "sanitize.css": "^13.0.0",
    "semver": "^7.5.3",
    "stylelint": "^15.9.0",
    "stylelint-config-html": "^1.1.0",
    "stylelint-config-prettier": "^9.0.4",
    "stylelint-config-recommended-vue": "^1.4.0",
    "stylelint-config-standard": "^33.0.0",
    "stylelint-order": "^6.0.3",
    "stylelint-plugin-logical-css": "^0.12.0",
    "ts-node": "^10.9.1",
    "typescript": "^5.0.4",
    "unplugin-auto-import": "^0.16.3",
    "unplugin-svg-component": "^0.7.3",
    "unplugin-vue-components": "^0.25.1",
    "vite": "^4.3.9",
    "vite-aliases": "^0.11.2",
    "vite-plugin-checker": "^0.6.0",
    "vite-plugin-dts": "^3.0.0-beta.1",
    "vite-plugin-fonts": "^0.7.0",
    "vite-plugin-inspect": "^0.7.29",
    "vite-plugin-mkcert": "^1.15.0",
    "vite-plugin-restart": "^0.3.1",
    "vite-plugin-vue-css-modules": "^3.1.2",
    "vite-svg-loader": "^4.0.0",
    "vitest": "^0.32.0",
    "vue": "^3.3.4",
    "vue-axios": "^3.5.2",
    "vue-clipboard3": "^2.0.0",
    "vue-demi": "^0.14.5",
    "vue-eslint-parser": "^9.3.1",
    "vue-i18n": "^9.2.2",
    "vue-router": "4.2.2",
    "vue-tsc": "^1.8.2",
    "vue3-cookies": "^1.0.6"
  },
  "devDependencies": {
    "@types/eslint": "^8.40.0",
    "@types/lodash": "^4.14.195",
    "@types/node": "^20.3.2",
    "@types/postcss-reporter": "^7",
    "@types/postcss-url": "^10",
    "@types/prettier": "^2.7.3"
  },
```

**Comment by Jevon617** [comment]
@thisVioletHydra It fixed at 0.7.4.
